packages/flutter/lib/src/material/stepper.dart

```diff
@@ -799,7 +803,7 @@ class _StepperState extends State<Stepper> with TickerProviderStateMixin {
         Material(
           elevation: widget.elevation ?? 2,
           child: Container(
-            margin: const EdgeInsets.symmetric(horizontal: 24.0),
+            margin: widget.stepPadding,
```

Comment:
The default should be introduced here: `widget.stepPadding ?? const EdgeInsets.all(24.0)`